CreeDictionary/tests/utils_tests/test_paradigm_filler.py

```diff
@@ -20,111 +21,25 @@ def test_import_prefilled_layouts(shared_datadir) -> None:
-            WordClass.NA,
-            ParadigmSize.FULL,
-            [
-                [
-                    [Label(text="One"), "niska", "", "", ""],
-                    [Label(text="Many"), "niskak", "", "", ""],
-                    [Label(text="Further"), "niska", "", "", ""],
-                    [Label(text="In/On"), "niskihk", "", "", ""],
-                    [Label(text="Among"), "niskinâhk", "", "", ""],
-                ],
-                [
-                    TitleRow(title="Smaller/Lesser/Younger", span=5),
-                    [Label(text="One"), "niskis", "", "", ""],
-                ],
-                [
-                    TitleRow(title="Ownership", span=5),
-                    [
-                        "",
-                        Heading(text="One"),
-                        Heading(text="Many"),
-                        Heading(text="Further"),
-                        Heading(text="In/On"),
-                    ],
-                    [
-                        Label(text="my"),
-                        "niniskim",
-                        "niniskimak",
-                        "niniskima",
-                        "niniskimihk",
-                    ],
-                    [
-                        Label(text="your (one)"),
-                        "kiniskim",
-                        "kiniskimak",
-                        "kiniskima",
-                        "kiniskimihk",
-                    ],
-                    [Label(text="his/her"), "", "", "oniskima", "oniskimihk"],
-                    [
-                        Label(text="our (but not your)"),
-                        "niniskiminân",
-                        "niniskiminânak",
-                        "niniskiminâna",
-                        "niniskiminâhk",
-                    ],
-                    [
-                        Label(text="your and our"),
-                        "kiniskiminaw",
-                        "kiniskiminawak",
-                        "kiniskiminawa",
-                        "kiniskiminâhk",
-                    ],
-                    [
-                        Label(text="your (all)"),
-                        "kiniskimiwâw",
-                        "kiniskimiwâwak",
-                        "kiniskimiwâwa",
-                        "kiniskimiwâhk",
-                    ],
-                    [Label(text="their"), "", "", "oniskimiwâwa", "oniskimiwâhk"],
-                    [
-                        Label(text="his/her/their (further)"),
-                        "",
-                        "",
-                        "oniskimiyiwa",
-                        "oniskimiyihk",
-                    ],
-                ],
-            ],
-        )
-    ],
-)
-def test_fill_NA_Full(
-    paradigm_filler, lemma, inflection_category, paradigm_size, expected_table
-):
-    assert (
-        paradigm_filler.fill_paradigm(lemma, inflection_category, paradigm_size)
-        == expected_table
-    )
-
-
```

Comment:
this test is too rigid, and it overlaps by this other test in `model_test.py` 

https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/pull/506/files#diff-952b874ad913920e9b450c364710c312R192-R200